src/dealDashboard/dealVotes/dealVotes.html

```diff
@@ -5,35 +5,47 @@
     <div class="content">
       <h1 class="title heading heading3 gradient">Vote</h1>
       <p class="bodySmallText">
-        You’re eligible to vote. If you choose not to vote, you will automatically reject the deal.
+        ${statusText.voteText}
       </p>
-      <pbutton-group>
-        <pbutton class="acceptButtonActive" full-width type="formfield">
+      <pbutton-group if.bind="ethereumService.defaultAccountAddress && deal.representativeDAO(ethereumService.defaultAccountAddress)">
```

Comment:
nit:
If the first bool needed? As I see it, deal.representativeDAO() checks that implicitly.

Dunno, maybe good to have the explicit first bool